lib/ruby_lsp/requests/inlay_hints.rb

```diff
@@ -36,13 +36,14 @@ def initialize(range, emitter, message_queue)
         @response = T.let([], ResponseType)
         @range = range
 
-        emitter.register(self, :on_rescue)
+        emitter.register(self, :on_node)
       end
 
-      sig { params(node: SyntaxTree::Rescue).void }
-      def on_rescue(node)
-        exception = node.exception
-        return unless exception.nil? || exception.exceptions.nil?
+      sig { params(node: T.nilable(YARP::Node)).void }
+      def on_node(node)
```

Comment:
Since YARP doesn't support `visit_rescue`, I'm not sure if there's a better way to do this?

Reply:
@vinistock reading https://github.com/ruby/yarp/blob/47fe0daddf2a2eab765e6b3ebff744d3fa614786/lib/yarp.rb#L152-L155, it sounds like we would need to implement a number of `visit_` methods, is that the approach we want to take?